.github/workflows/platform.dependencies.yml

```diff
@@ -915,4 +915,4 @@ jobs:
           resourceGroupName: '${{ env.resourceGroupName }}'
           subscriptionId: '${{ secrets.ARM_SUBSCRIPTION_ID }}'
           managementGroupId: '${{ secrets.ARM_MGMTGROUP_ID }}'
-          removeDeployment: '${{ env.removeDeployment }}'
+          removeDeployment: '${{ env.removeDeployment }}'
```

Comment:
Was that intended?

Reply:
Unintended. Restored the file to remove any changes.